src/fides/data/privacy_notices/privacy_notice_templates.yml

```diff
@@ -5,39 +5,39 @@ privacy_notices:
   description: This website uses essential cookies and services to enable core website features and provide a seamless user experience. These cookies and services are used to facilitate features such as navigation, remember user preferences, and ensure the security of the website.
   internal_description: Essential cookies and services, also known as strictly necessary, are technologies that are crucial for the proper functioning of a website. These technologies are typically exempt from requiring user consent under privacy regulations because they are essential for the website to deliver its basic functionalities and services.
   regions:
-    - eu_be
-    - eu_bg
-    - eu_cz
-    - eu_dk
-    - eu_de
-    - eu_ee
-    - eu_ie
-    - eu_el
-    - eu_es
-    - eu_fr
-    - eu_hr
-    - eu_it
-    - eu_cy
-    - eu_lv
-    - eu_lt
-    - eu_lu
-    - eu_hu
-    - eu_mt
-    - eu_nl
-    - eu_at
-    - eu_pl
-    - eu_pt
-    - eu_ro
-    - eu_si
-    - eu_sk
-    - eu_fi
-    - eu_se
+    - be
+    - bg
+    - cz
+    - dk
+    - de
+    - ee
+    - ie
+    - el
+    - es
+    - fr
+    - hr
+    - it
+    - cy
+    - lv
+    - lt
+    - lu
+    - hu
+    - mt
+    - nl
+    - at
+    - pl
+    - pt
+    - ro
+    - si
+    - sk
+    - fi
+    - se
     - gb_eng
     - gb_sct
     - gb_wls
     - gb_nir
-    - isl
-    - nor
+    - is
+    - "no"
```

Comment:
Both "is" and "no" were problematic when I first put this in, so I used "isl" and "nor" instead.  I knew that "is" was failing because it was a python keyword, but "no" was actually failing in the yaml, not python because it was being transformed to a boolean. Quoting this helps.